github/ipfs.yml

```diff
@@ -4958,6 +4964,21 @@ teams:
         - laurentsenta
     parent_team_id: w3dt-stewards
     privacy: closed
+  # NOTE: created to capture users with push+ access to github-mgmt repository
+  org-admins:
+    create_default_maintainer: false
+    description: Users that are effectively org admins
+    # WARN: membership here should be treated exactly as cautiosly as having an org admin role
+    # ATTN: members are expected to:
+    #  - be familiar with GitHub Management 
+    #  - be ready to triage/review org configuration change request in github-mgmt
+    members:
+      maintainer:
+        - biglep
```

Comment:
Maybe make it so I am just a member so I too am forced to use github management?  
We still have the escape hatch of true admins can use the UI

Reply:
As long as you're an actual org admin, then you're a maintainer of all the teams you belong to. GitHub API will happily accept a request to change your role to `member` but it doesn't really do anything and a subsequent query for your role will still return `maintainer`.

Same goes for Adin and Lidel so I moved them up here too.